src/components/ShellView/ResultGraph.vue

```diff
@@ -453,10 +511,16 @@ export default {
           strength: 2,
         },
         collide: {
-          radius: 40,
+          radius: 120,
+        },
+        manyBody: {
+          strength: -100,
         },
-        alphaMin: 0.2,
+        alpha: 1,
+        alphaMin: 0.02,
         alphaDecay: 0.03,
+        velocityDecay: 0.6,
+    
```

Comment:
Let's undo this for now. I can see it results in more relaxed layout, but can also signigicantly increase the time to converge. I tested on scale of 1000 nodes and the render time increases from 8.3 secs to 15.4 secs. Maybe we should consider providing options or adjust based on scale.